cruise-control-client/cruisecontrolclient/client/CCParameter/BooleanParameter.py

```diff
@@ -185,5 +186,5 @@ class VerboseParameter(AbstractBooleanParameter):
     argparse_properties = {
         'args': ('--verbose',),
         # There is no sensible default here; allow for ternary state
-        'kwargs': dict(help=description, type=bool, metavar='BOOL')
+        'kwargs': dict(help=description, choices=set_of_choices, metavar='BOOL')
```

Comment:
Are (1) `allow_capacity_estimation`, (2) `skip_rack_awareness_check`, and (3) `dryrun` intentionally inconsistent with the rest of the requests?

Reply:
Yes this is intentional.

The reasoning here is that there is a sensible default value for each of these parameters (namely, `false`) which is explicitly passed by `cccli` to `cruise-control` whenever the appropriate endpoint is called.

I am aware that this effectively shadows existing `cruise-control` server-side defaults.